changelog.d/.towncrier_template.md.jinja

```diff
@@ -0,0 +1,57 @@
+*{{ versiondata["date"] }}*
```

Comment:
Don't we have to render the actual version title? Looks like it's missing. Never mind the `sphinxcontrib-towncrier` preview. It may be quirky.

Reply:
I think that old versions of Towncrier would prepend the version title outside Jinja and than it changed. The recent Sphinx plugin version doesn't support those old versions of Towncrier anymore.
Also, the plugin adds a title IIRC, needs checking but I wouldn't worry about the preview. Making sure that the regular release run puts it in place is what's most important.